underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -350,17 +350,24 @@ private String convertToFolderName(String key) {
    * @return true if the operation was successful, false otherwise
    */
   private boolean copy(String src, String dst) {
-    try {
-      src = stripPrefixIfPresent(src);
-      dst = stripPrefixIfPresent(dst);
-      LOG.info("Copying {} to {}", src, dst);
-      GSObject obj = new GSObject(dst);
-      mClient.copyObject(mBucketName, src, mBucketName, obj, false);
-      return true;
-    } catch (ServiceException e) {
-      LOG.error("Failed to rename file {} to {}", src, dst, e);
-      return false;
+    src = stripPrefixIfPresent(src);
+    dst = stripPrefixIfPresent(dst);
+    LOG.info("Copying {} to {}", src, dst);
+    GSObject obj = new GSObject(dst);
+    int retries = 3;
+    for (int i = 0; i < retries; ++i) {
```

Comment:
This is my habit from c++... Google c++ style guide prefers `++i `over `i++`: https://google.github.io/styleguide/cppguide.html#Preincrement_and_Predecrement

In c++, `++i` can have slightly better performance than `i++` in some cases: http://stackoverflow.com/questions/24901/is-there-a-performance-difference-between-i-and-i-in-c

Though it might not matter in java. Do we have convention to use `i++` over `++i` in Alluxio?